Security/Http/Firewall/LegacyTwoFactorListener.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Security\Http\Firewall;
+
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\Security\Http\Firewall\ListenerInterface;
+
+class LegacyTwoFactorListener implements ListenerInterface
+{
+    private $twoFactorListener;
+
+    public function __construct(TwoFactorListener $twoFactorListener)
+    {
+        $this->twoFactorListener = $twoFactorListener;
+    }
+
+    public function handle(GetResponseEvent $event)
+    {
+        $this->twoFactorListener->__invoke($event);
```

Comment:
Let's better use `($this->twoFactorListener)($event);` to be aligned with what we do in the tests and not directly call a magic method.